starters/oasp4j-security-starter/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.oasp.java.dev</groupId>
+    <artifactId>oasp4j-starters</artifactId>
+    <version>dev-SNAPSHOT</version>
+  </parent>
+  <groupId>io.oasp.java.starters</groupId>
+  <artifactId>oasp4j-security-starter</artifactId>
+  <version>${oasp4j.version}</version>
+  <packaging>jar</packaging>
+  <name>${project.artifactId}</name>
+  
+  <dependencies>
+    <dependency>
+      <groupId>${oasp.modules.groupId}</groupId>
+      <artifactId>oasp4j-security</artifactId>
+      <version>${oasp4j.version}</version>
```

Comment:
We are using a BOM. Why do we have to add version's here at all? Please remove versions and properly use BOM.